### ISSUE TITLE
Common functions

### ISSUE THREAD

**Issue by ravi160822** [sympy/sympy#24364]
Is there a plan to add common loss functions like MSE, CrossEntropy to sympy.
Including functions like softmax.

so one could do something like,
```
from sympy import softmax
```
and see the softmax equation, along with adding additional arguments required for softmax

**Comment by oscarbenjamin** [comment]
I don't know if anyone plans to add these things or what they would be used for if they were added. If you think that there are good reasons to add them then please:

1. Explain precisely what these functions are.
2. Explain precisely how these might work in SymPy.
3. Describe what they might be used for.
4. Explain why there is a benefit in adding them rather than using the existing capabilities of SymPy.

The idea that someone imports the softmax function just to "see" an equation is not a convincing argument for adding these to SymPy. Any mathematical function added to SymPy entails a long tail of maintenance work and bug fixes and there needs to be a nontrivial benefit for that to be worthwhile.

Here are two pull requests in the past that are related to the sigmoid function in the past: https://github.com/sympy/sympy/pull/15952 https://github.com/sympy/sympy/pull/20758#event-4212552829

Those two pull requests were sort of rejected. I say sort of because essentially what happened is the authors proposed something and then gave up when challenged to give a good motivation for the proposed changes. Likewise here I suggest closing this unless a significant motivation is given.
